app/libs/full_contact_company_processor.rb

```diff
@@ -0,0 +1,60 @@
+class FullContactCompanyProcessor
+  def initialize(company:)
+    @company = company
+  end
+
+  def process
+    @response = call_fullcontact_api
+    return if @response.nil? || @response['status'] != 200
+    process_links
+    process_organization
+    set_industries
+    @company.save
+    @company
+  end
+
+  private
+
+  def process_links
+    @company.twitter = find_url('twitter')
+    @company.facebook = find_url('facebook')
+    @company.linkedin = find_url('linkedincompany')
+    @company.youtube = find_url('youtube')
+    @company.angellist = find_url('angellist')
+    @company.owler = find_url('owler')
+    @company.crunchbasecompany = find_url('crunchbasecompany')
+    @company.pinterest = find_url('pinterest')
+    @company.google = find_url('google')
+    @company.klout = find_url('klout')
+  end
+
+  def process_organization
+    organization = @response['organization']
+    @company.name = organization['name'] || ''
```

Comment:
remove " || ''" from everywhere
if you want to have this fields not nil - add in migration default value = ''